File: entorno/Server1/MySite/Files/Pagina.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApi.Models.AB
{
    public class Pagina
    {

        //=========

        public Pagina[] Ramas = new Pagina[5];
        public Nodo[] Claves = new Nodo[5];
        public int Cuentas = 0;
        public Queue<String> Hijos = new Queue<string>();

        public Pagina(Nodo clave)
        {
            Claves[0] = clave;
        }

        public Pagina()
        {
        }

        public Queue<String> hijos
        {
            get { return Hijos; }
            set { Hijos = value; }

        }
    }
}

File: entorno/Server1/MySite/Files/Pagina.py
from collections import deque


class Pagina(object):

	def __init__(self):
		self.Ramas = [None, None,None,None,None]
		self.Claves = [None, None,None,None,None]
		self.Cuentas = 0 #es el numero de claves llenas
		self._Hijos = deque()
		self.name = ""
		print 'se creo una nueva pagina'


	def get_hijos(self):
		return self._Hijos

	def set_hijos(self, value):
		self._Hijos = value

	hijos = property(fget=get_hijos, fset=set_hijos)



class Nodo(object):

	def __init__(self,id):
		self._carpetas = None
		self._archivos = None
		self.Id = id

	def addCarpeta(self):
		print 'add'